site/en/docs/extensions/reference/sidePanel/index.md

```diff
@@ -0,0 +1,202 @@
+---
+api: sidePanel
+---
+
+## Overview {: #overview }
+
+Chrome features a built-in side panel that enables users to view more information alongside the main content of a webpage. The SidePanel API allows extensions to add their own panels to the Chrome browser. Some features include:
+
+- The side panel remains open when navigating between tabs (if set to do so).
+- It can be available only on specific websites.
+- As an extension page, side panels have access to all Chrome APIs.
+- Users can choose which side to display the panel under Chrome settings.
+
+## Manifest {: #manifest }
+
+To use the SidePanel API, add the `"sidePanel"` permission in the extension [manifest][doc-manifest] file:
+
+{% Label %}manifest.json:{% endLabel %}
+
+```json
+{
+  "name": "My side panel extension",
+  ...
+  "permissions": [
+    "sidePanel"
+  ]
+}
+```
+
+## Use cases {: #use-cases }
+
+The following sections demonstrate some common use cases for the SidePanel API. See [Extension samples](#examples) for complete extension examples.
+
+### Display the same side panel on every site {: #every-site }
+
+The side panel can be set initially from the `“default_path”` property in the `“side_panel”` key of the manifest to display the same side panel on every site. This should point to a relative path within the extension directory.
```

Comment:
I think something might've messed up your quotes here?

```suggestion
The side panel can be set initially from the `"default_path"` property in the `"side_panel"` key of the manifest to display the same side panel on every site. This should point to a relative path within the extension directory.
```

Reply:
Gah 🫢, thanks for catching that!